pkg/fake/utils.go

```diff
@@ -155,3 +160,66 @@ func matchTags(tags []*ec2.Tag, filter *ec2.Filter) bool {
 	}
 	return false
 }
+
+func MakeFakeInstances() []*ec2.InstanceTypeInfo {
```

Comment:
```suggestion
func MakeInstances() []*ec2.InstanceTypeInfo {
```

Do you need fake in the name of any of these packages anymore since the call will always be prefixed with "fake". It's more idiomatic to say `fake.MakeInstances()`. The same applies to the rest of the functions that are shifting to this package

Reply:
Changed. Makes sense.